src/server/scripts/BrokenIsles/Dalaran/VioletHold/violet_hold_broken_isles.cpp

```diff
@@ -86,6 +61,96 @@ enum GameObjectId
     GO_PRISON_SEAL = 247002
 };
 
+enum CreatureId
+{
+    NPC_SUMMON_PORTAL = 102267,
+    NPC_FELSTALKER_RAVENER = 102269,
+    NPC_EREDAR_INVADER = 102270,
+    NPC_FELGUARD_DESTROYER = 102272,
+    NPC_DOOMGUARD_INFILTRATOR = 102273
+};
+
+Position const TempSummonPositions[3] =
+{   //     X         Y           Z         O
+    { 4623.08f, 4059.6f, 82.7241f, 4.7822f }, // 1 Portal Left
+    { 4685.84f, 4013.8f, 91.8186f, 3.12414f }, // 2 Portal Middle
+    { 4638.16f, 3962.08f, 87.0574f, 1.71042f }, // 3 Portal Right
+};
+
+
+class npc_summon_portal : public CreatureScript
+{
+public:
+    npc_summon_portal() : CreatureScript("npc_summon_portal") { }
+
+    struct npc_summon_portalAI : public ScriptedAI
+    {
+        npc_summon_portalAI(Creature* creature) : ScriptedAI(creature), Summons(me) { }
+
+        void Reset() override
+        {
+            _events.Reset();
+            Summons.DespawnAll();
+        }
+
+        void JustAppeared() override
+        {
+            _events.ScheduleEvent(EVENT_SUMMONS, 20s);
+        }
+
+        void JustSummoned(Creature* summon) override
+        {
+            Summons.Summon(summon);
+
+            switch (summon->GetEntry())
+            {
+            case NPC_FELSTALKER_RAVENER:
+            case NPC_EREDAR_INVADER:
+            case NPC_FELGUARD_DESTROYER:
+            case NPC_DOOMGUARD_INFILTRATOR:
+                summon->RemoveAura(29266);
```

Comment:
This is because you probably got an update on creature_template_addon for several of these creatures because they're the same ones fighting and dying at the beginning of the dungeon before portals are initialized. This is why you cannot simply rely on the parser to change stuff without reviewing. 

You need to remove this RemoveAura and make sure the specific spawns that are dead have this aura on creature_addon.

Reply:
you can safely remove that part, its done here: https://github.com/TrinityCore/TrinityCore/commit/bbd4ed788a6d774fa52cadf1321929f822df6414#diff-b379cda0dcf15914e554b2fbb35649984ddc44f1c53873b8cf4e86e02b700cb0R118